tests/test_fastqc.py

```diff
@@ -19,12 +19,17 @@ def test_fastqc():
     test_dir = "temp_test_fastqc"
     os.makedirs(f"./{test_dir}", exist_ok=True)
     output_dir_path = f"./{test_dir}"
-
+    filtered_html_output_path = f"{output_dir_path}/sample_r1_shortened_fastqc_filtered.html"
     toolchest.fastqc(
         inputs="s3://toolchest-integration-tests/sample_r1_shortened.fastq",
         output_path=output_dir_path
     )
 
-    assert hash.unordered(os.path.join(test_dir, "sample_r1_shortened_fastqc.html")) == 1123268405
-
-    assert os.path.getsize(os.path.join(test_dir, "sample_r1_shortened_fastqc.zip")) == 326078
+    filter_output.filter_regex(
+        os.path.join(test_dir, "sample_r1_shortened_fastqc.html"),
+        filtered_html_output_path,
+        search_regex='id="header_filename">([\\w\\s]+)<br',
+        replacement_str='id="header_filename">)<br'
```

Comment:
Correct me if I'm wrong, but doesn't this drop the left paren `(` as well as the date? (I'm presuming the date is outputted in between the parens.) 

It shouldn't matter and doesn't change the functionality of the test, so I'm fine leaving this as is.

Reply:
The date is actually in between the html tags. The paren was missed when I was deleting the regex group from the search string to replace with the same html minus the group. I'll push a new version up with that fixed.